Description: Allow routes 2.0
 Upstream is rejecting Routes 2.0 because the archive on PyPi is broken,
 however, the Debian package for Routes 2.0 is perfectly fine. So we remove
 the restriction.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-03-19

--- heat-2015.1~b2+git20150318+2956841a2e.orig/requirements.txt
+++ heat-2015.1~b2+git20150318+2956841a2e/requirements.txt
@@ -39,7 +39,7 @@ python-troveclient>=1.0.7
 PyYAML>=3.1.0
 qpid-python
 requests>=2.2.0,!=2.4.0
-Routes>=1.12.3,!=2.0
+Routes>=1.12.3
 six>=1.9.0
 SQLAlchemy>=0.9.7,<=0.9.99
 sqlalchemy-migrate>=0.9.5
